src/qt/network.cpp

```diff
@@ -37,19 +39,18 @@
 #pragma GCC diagnostic pop
 
 #include "utils.h"
-#include "prices.h"
 
-Prices::Prices(QObject *parent)
+Network::Network(QObject *parent)
     : QObject(parent)
     , m_scheduler(this)
 {
 }
 
-void Prices::getJSON(const QString url) const
+void Network::get(const QString url, const QJSValue& callback, const QString &content_type) const
 {
     qDebug() << QString("Fetching: %1").arg(url);
 
-    m_scheduler.run([this, url] {
+    m_scheduler.run([this, url, content_type] {
```

Comment:
```suggestion
    m_scheduler.run([url, content_type] {
```

See macOS CI, fails due to werror

Reply:
Done